components/pages/Applications/ApplicationForm/Forms/Collaborators/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { createRef, ReactElement, useCallback, useEffect, useState } from 'react';
+import React, { createRef, ReactElement, useCallback, useEffect, useState } from 'react';
```

Comment:
Not needed in React 17+
```suggestion
import { createRef, ReactElement, useCallback, useEffect, useState } from 'react';
```